src/metabase/api/routes.clj

```diff
@@ -72,7 +72,7 @@
   (context "/activity"                    [] (+auth activity/routes))
   ;; order is important here! the /public version must appear first, otherwise
   ;; it does not apply.
-  (context "/advanced_computation/public" [] (+generic-exceptions advcomp-public/routes))
+  (context "/advanced_computation/public" [] (+message-only-exceptions advcomp-public/routes))
```

Comment:
I don't think it makes sense to to have the public *and* embed versions of the pivot table endpoints be under `/api/advanced_computation/public` when *all* of our other public/embed endpoints are just `/api/public/original/endpoint` and `/api/embed/original/endpoint`, respectively. I know at some point the FE client just changed the endpoint prefix from something like `http://localhost:300/api/` to `http://localhost:3000/api/public` or `http://localhost:3000/api/embed` for public/embedded stuff and since the endpoint paths are otherwise the same it would work without issue. Not sure if this is still the case but it's nice to use a consistent pattern for our public/embed versions of things and to be able to tell if an API request is public or embedding by looking at the beginning of the route.

IMO it's nice to have _all_ the public and embed stuff in the two respective namespaces so we can keep an eye on what's in there and reuse the same tooling when we have to add new stuff. I try to write the "normal" versions of the endpoints in a way that lets me make simple wrappers for them in `public` and `embed` -- see the chain filtering endpoints I added recently for example.  

If there's a reason why the new endpoints can't go in the same namespaces as all the other public or embed stuff, I would call them something like `metabase.api.public.advanced-computation` and `metabase.api.embed.advanced-computation`